src/main/java/sirius/biz/storage/versions/VersionedFile.java

```diff
@@ -15,7 +15,7 @@
 /**
  * Entity holding meta information about a versioned file.
  */
-@Framework(Storage.FRAMEWORK_STORAGE)
+@Framework(SQLBlobStorage.FRAMEWORK_JDBC_BLOB_STORAGE)
```

Comment:
Is there no Mongo implementation for this or in other words is no mongo version planned as this has a generic name and lives in a generic package?
This does not match the other biz/db stuff.

Reply:
This is a legacy framework which was only built for SQL - this will be refactored, in the meantime we bind this to the sql storage.